client/apis/clients.js

```diff
@@ -3,9 +3,10 @@ import request from 'superagent'
 
 const rootUrl = '/api/v1/clients'
 
-// export function getAllClients() {
-//   return request.get(rootUrl).then((res) => {
-//     // console.log(res.body)
-//     return res.body
-//   })
-// }
+export function getClients(filter) {
+  return request.get(rootUrl + '?filter=' + filter).then((res) => {
+    return res.body
+  })
+}
+
+getClients('active')
```

Comment:
Why is getClients's being called in the API file?

Reply:
Left overs from when we built that api client function. Cleaning it up